### PR TITLE
add class pad to timeslider to fix height issue

### LINKED ISSUE #4937: Timeslider does not work
**Describe the bug**
Timeslider page works but it does not show the contents of the pad like it used to
Console has an error:
 Unexpected error: could not translate element content for key timeslider.toolbar.returnbutton

**To Reproduce**
1. Create pad
2. edit
3. go to timeslider
4. See error in console

**Expected behavior**
Timeslider should show the edits of the pad

**Screenshots**
screen

**Server (please complete the following information):**
 - Etherpad version: clone from develop branch today
 - OS: docker node:10-buster-slim:lastest
 - Node.js version (`node --version`): docker node:10-buster-slim:lastest
 - npm version (`npm --version`): docker node:10-buster-slim:lastest
 - db: 10.4.18-MariaDB

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Firefox or any other
 - Version Latest

### PR DESCRIPTION
**Pull request by webzwo0i**
instead of https://github.com/ether/etherpad-lite/pull/4939

let's see what frontend test output is

Fixes #4937

### CODE REVIEW

File: src/static/css/pad/layout.css [hunk 1]
```diff
@@ -4,6 +4,14 @@ html, body {
   margin: 0;
   padding: 0;
 }
+
+html.timeslider {
```
Comment (rhansen) [inline src/static/css/pad/layout.css:8]: To be 100% consistent with what it was before commit 470f40d7db01b68856edf47ac7a5befd50b50340:

```suggestion
html.timeslider, html.timeslider body {
```

**Review by rhansen — APPROVED** [review]

File: src/static/css/pad/layout.css [hunk 2]
```diff
@@ -4,6 +4,14 @@ html, body {
...
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
```
Comment (rhansen) [inline src/static/css/pad/layout.css:12]: width, margin, and padding are already covered by the above `body, html {}` block:

```suggestion
  height: 100%;
```

File: src/templates/timeslider.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
     , langs = require("ep_etherpad-lite/node/hooks/i18n").availableLangs
 %>
 <!doctype html>
-<html class="<%=settings.skinVariants%>">
+<html class="timeslider <%=settings.skinVariants%>">
```
Comment (rhansen) [inline src/templates/timeslider.html:6]: If we use `pad` instead then we can reuse the existing CSS rules:

```suggestion
<html class="pad <%=settings.skinVariants%>">
```

**Review by rhansen — APPROVED** [review]

**Comment by webzwo0i** [comment]
thanks for review! Now all it does is adding the "pad" class to timeslider's HTML

**Pull request merged**